_guide/_pages/interviews/code-review.md

```diff
@@ -46,36 +46,37 @@ For the list of things to look for during code review, see [what to look for dur
 
 Begin by introducing yourself by saying this or something similar to it:
 
-> Hello! My name is \_\_\_, my pronouns are \_\_\_, and my role at 18F is \_\_\_.
+> Hello! My name is \_\_\_, my pronouns are \_\_\_, and my role at TTS is \_\_\_.
```

Comment:
Another place where TTS/18F comes up — maybe "and my role is ____" would be sufficient.